extras/travis_build.sh

```diff
@@ -117,7 +117,7 @@ echo $NEST_VPATH
 mkdir "$NEST_VPATH" "$NEST_RESULT"
 mkdir "$NEST_VPATH/reports"
 
-if [ "$xSTATIC_ANALYSIS" = "1" ]; then
+if [ "$xSTATIC_ANALYSIS" == "1" ]; then
```

Comment:
The script uses 3 different variants of the bash conditionals syntax.
if [ "$xSTATIC_ANALYSIS" = "1" ]; then ..
if [ "$xSTATIC_ANALYSIS" == "1" ]; then ..
if [[ "$xSTATIC_ANALYSIS" == "1" ]]; then ..
The first one, I believe, is the standard.

Reply:
There are subtle differences (especially between `[]` and `[[]]`), and the only reason I didn't address this before is that I'm afraid to break things. E.g. `if [[ $OSTYPE == darwin* ]]` would break if it was changed for single brackets.

Reply:
If I'm right, **==** is an alias for **=** and **[[ ... ]]** allows to omit the quotes while empty strings are handled properly. However, I'm fine with all variants. The point I wanted to make is that for the same thing we should use the same syntax, i.e. when checking the build-matrix variables. Of course, I would leave it as it is for the OSTYPE.